cpu/kinetis_common/doc.txt

```diff
@@ -185,126 +177,120 @@
+ *                          .pin_cs   = {
+ *                              GPIO_PIN(PORT_E, 4),
+ *                              GPIO_UNDEF,
+ *                              GPIO_UNDEF,
+ *                              GPIO_UNDEF,
+ *                              GPIO_UNDEF
+ *                          },
+ *                          .pcr      = GPIO_AF_2,
+ *                          .simmask  = SIM_SCGC6_SPI1_MASK
+ *                      }
+ *                  };
+ *
+ *                  #define SPI_NUMOF           (sizeof(spi_config) / sizeof(spi_config[0]))
  */
 
 /**
  * @defgroup    cpu_kinetis_common_timer Kinetis Timer
  * @ingroup     cpu_kinetis_common
- * @brief       Periodic Interrupt Timer (PIT) driver.
- *              Implementation of riot-os low level timer interface
- *              for the Kinetis Periodic Interrupt Timer.
+ * @brief       Periodic Interrupt Timer (PIT) and Low-Power Timer (LPTMR) driver.
+ *
  *              The PIT is a count down timer, in order to use it with riot-os
  *              a count up timer will be simulated. The PIT has four channels,
  *              each two channels are cascaded. The n-1 channel is a prescaler
  *              and the n channel a down counter. In standard configuration
  *              with four channels, two simulated count up timer are possible.
  *
- *              ### Timer configuration Example (for periph_conf.h) ###
+ *              To counteract the effects of the asynchronous operation of the
+ *              LPTMR, this driver uses the RTT as a time base which the LPTMR
+ *              is referenced against. This method reduces the timing jitter
+ *              caused by mixing the clock domains of the bus clock used by the
+ *              CPU and the 32kHz reference clock for the LPTMR counter.
  *
- *                  #define TIMER_NUMOF             (1U)
- *                  #define TIMER_0_EN              1
- *                  #define TIMER_1_EN              0
- *                  #define TIMER_IRQ_PRIO          1
- *                  #define TIMER_BASE              PIT
- *                  #define TIMER_MAX_VALUE         (0xffffffff)
- *                  #define TIMER_CLOCK             CLOCK_CORECLOCK
- *                  #define TIMER_CLKEN()           (SIM->SCGC6 |= (SIM_SCGC6_PIT_MASK))
- *
- *                  // Timer 0 configuration
- *                  #define TIMER_0_PRESCALER_CH    0
- *                  #define TIMER_0_COUNTER_CH      1
- *                  #define TIMER_0_ISR             isr_pit1
- *                  #define TIMER_0_IRQ_CHAN        PIT1_IRQn
- *
- *                  // Timer 1 configuration
- *                  #define TIMER_1_PRESCALER_CH    2
- *                  #define TIMER_1_COUNTER_CH      3
- *                  #define TIMER_1_ISR             isr_pit3
- *                  #define TIMER_1_IRQ_CHAN        PIT3_IRQn
+ *              ### Timer configuration Example (for periph_conf.h) ###
```

Comment:
(very) nit picky: should either be `Timer [C]onfiguration Example` or `Timer configuration [e]xample`